login.css

```diff
@@ -28,7 +28,7 @@ nav{
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
```

Comment:
08/08/2021: I need to rearrange the margin for them link in the nav because it was too big so 30px is the right amount.